pallets/parachain-staking/src/tests.rs

```diff
@@ -6618,22 +6616,6 @@ fn deferred_payment_steady_state_event_flow() {
 				set_author(round as BlockNumber, 4, 1);
 			};
 
-			// grab initial issuance -- we will reset it before round issuance is calculated so that
-			// it is consistent every round
-			let initial_issuance = Balances::total_issuance();
-			let reset_issuance = || {
-				let new_issuance = Balances::total_issuance();
-				let diff = new_issuance - initial_issuance;
-				let burned = Balances::burn(diff);
-				Balances::settle(
-					&111,
-					burned,
-					WithdrawReasons::FEE,
-					ExistenceRequirement::AllowDeath,
-				)
-				.expect("Account can absorb burn");
-			};
-
```

Comment:
What is the context for this change?

Reply:
The `burn()` signature changed, and It didn't make sense to me in this context, after this change https://github.com/paritytech/polkadot-sdk/pull/3964/files#diff-5dfe6adcc3c44280fc10cd49830df408a491c768a54f47835d822a0d1334d385R780

Reply:
We are defining a burn account (`111`) with `1000` initial balance above. I do not see why we should remove this logic from the test, lets keep it.

Reply:
You are right, I saw the function reset_issuance() wasn't causing a problem in the tests (not making them failed) so I removed it. But there's no a strong need to remove it. I did added it again, with the necessary refactor.